### PR TITLE
[XM] fixed warnings in the "SourceWriter" project

### PR DESCRIPTION
**Pull request by mykyta-bondarenko**
fixed the [bug](https://bugzilla.xamarin.com/show_bug.cgi?id=45772)

### CODE REVIEW

File: SourceWriter/SourceWriter/Main.storyboard [hunk 1]
```diff
@@ -1178,9 +1178,9 @@ Gw
                                         <action selector="OriginalSize:" target="n8A-zD-eLP" id="jHe-Df-YoN"/>
                                     </connections>
                                 </toolbarItem>
-                                <toolbarItem implicitItemIdentifier="NSToolbarPrintItem" explicitItemIdentifier="Print" id="WFE-hK-GLq" customClass="ManualToolbarItem">
+                                <toolbarItem implicitItemIdentifier="NSToolbarPrintItem" explicitItemIdentifier="PrintPreview" id="WFE-hK-GLq" customClass="ManualToolbarItem">
                                     <connections>
-                                        <action selector="print:" target="n8A-zD-eLP" id="NeO-eg-Fvs"/>
+                                        <action selector="printPreview:" target="n8A-zD-eLP" id="NeO-eg-Fvs"/>
```
Comment (chamons) [inline SourceWriter/SourceWriter/Main.storyboard]: Why this change needed?

Comment (mykyta-bondarenko-gl) [reply SourceWriter/SourceWriter/Main.storyboard]: Sorry, it was my mistake. Initially I thought that it was the reason of the warning.
